### ISSUE TITLE
Replace camelcase

### ISSUE THREAD

**Issue by jariji** [Seelengrab/PropCheck.jl#12]
The Julia style guide [says](https://docs.julialang.org/en/v1/manual/style-guide/#Use-naming-conventions-consistent-with-Julia-base/)

> functions are lowercase ([maximum](https://docs.julialang.org/en/v1/base/collections/#Base.maximum), [convert](https://docs.julialang.org/en/v1/base/base/#Base.convert)) and, when readable, with multiple words squashed together ([isequal](https://docs.julialang.org/en/v1/base/base/#Base.isequal), [haskey](https://docs.julialang.org/en/v1/base/collections/#Base.haskey)). When necessary, use underscores as word separators. Underscores are also used to indicate a combination of concepts ([remotecall_fetch](https://docs.julialang.org/en/v1/stdlib/Distributed/#Distributed.remotecall_fetch-Tuple{Any,%20Integer,%20Vararg{Any}}) as a more efficient implementation of fetch(remotecall(...))) or as modifiers.

Would you mind applying that here? It's easier to learn and use an API if it follows the same conventions as the rest of the ecosystem and doesn't stick out in my code.

That would apply to `shrinkTowards` and any others.

**Comment by Seelengrab** [comment]
Yeah, that can be done. I'm not happy with the current function names either, but noone complained about the names so far :shrug:  Is there a style checker that could be used here/run in CI to keep the convention consistent going forward? Otherwise, manual search & replace is doable too.

**Comment by jariji** [comment]
> Is there a style checker that could be used here/run in CI to keep the convention consistent going forward?

I don't know of one. `rg '\b[a-z]+[A-Z]\w+'` seems to find them, I imagine any editor can do the replacement.
